src/discussions/discussions-home/DiscussionsHome.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Route, Switch } from 'react-router';
+import { Routes } from '../../data/constants';
+import { BreadcrumbMenu, NavigationBar } from '../navigation';
+import { buildDiscussionsUrl } from '../utils';
+import PostsViewContainer from '../posts/PostsViewContainer';
+import TopicsViewContainer from '../topics/TopicsViewContainer';
+import CommentsViewContainer from '../comments/CommentsViewContainer';
+
+export default function DiscussionsHome() {
+  const unembeddedCommentsPath = buildDiscussionsUrl(Routes.COMMENTS.PATH, { embed: '' });
+  const embeddedPostViewPath = buildDiscussionsUrl(Routes.VIEWS.POST, { embed: 'embed' });
+  return (
+    <main className="container-fluid my-4">
+      <div className="d-flex flex-column">
+        <Switch>
+          <Route path={Routes.VIEWS.DISCUSSION}>
+            <NavigationBar />
+            <Switch>
+              <Route
+                path={[
+                  Routes.POSTS.PATH,
+                  Routes.TOPICS.CATEGORY,
+                  unembeddedCommentsPath,
+                ]}
+                component={BreadcrumbMenu}
+              />
+            </Switch>
+          </Route>
+        </Switch>
+        <div className="d-flex flex-row">
+          <Switch>
+            { /* Do not render any of these if the view is the embedded post view */}
+            <Route path={embeddedPostViewPath} />
+            <Route path="*">
```

Comment:
Unfortunately, it's not possible to pass a `negative` path. In other words, you can only specify what you want to match against, not what you want to avoid. As such, the only way to NOT render something for a specific path is to render it when it is not that path. That is why these routes are setup this way